test/run-nools-lib.js

```diff
@@ -3,31 +3,36 @@ const taskEmitter = require('../src/nools/task-emitter');
 
 const { TEST_DATE } = require('./nools/mocks.js');
 
-const runNoolsLib = ({ c, targets, tasks }) => {
+const runNoolsLib = ({ c, targets, tasks, utilsMock }) => {
   const emitted = [];
   const context = {
     now: new Date(TEST_DATE),
     c,
     targets,
     tasks,
-    Utils: {
-      addDate: function(date, days) {
-        const d = new Date(date.getTime());
-        d.setDate(d.getDate() + days);
-        d.setHours(0, 0, 0, 0);
-        return d;
-      },
-      now: () => new Date(TEST_DATE),
-      isTimely: function() { return true; },
+    Utils: () => {
+      if (utilsMock) {
+        return utilsMock;
+      }
+      return {
```

Comment:
You can add a comment or (ToDo comment) explaining that the prefer way is to use 'utilsMocks' and this one will be removed when the unit test are alined with the new approach.
See comment below